Makefile

```diff
@@ -496,23 +495,25 @@ ifneq ($(shell mdbook --version > /dev/null 2>&1 && echo $$?), 0)
 	@echo -e "    cargo +stable install mdbook-linkcheck --force"
 	@exit 1
 endif
-	@mdbook build $(BOOK_SRC)
-	@rm -rf $(BOOK_OUT)
-	@mkdir -p $(BOOK_OUT)
-	@cp -rf $(BOOK_SRC)/book/html/* $(BOOK_OUT)
+	@mdbook build $(BOOK_SRC) -d $(BOOK_OUT)
 	@echo -e "\nThe Theseus Book is now available at \"$(BOOK_OUT_FILE)\"."
 
 
 ### Opens the Theseus book.
 view-book: book
 	@echo -e "Opening the Theseus book in your browser..."
 ifneq ($(IS_WSL), )
-	wslview "$(shell realpath --relative-to="$(ROOT_DIR)" "$(BOOK_OUT)")" &
+	wslview "$(shell realpath --relative-to="$(ROOT_DIR)" "$(BOOK_OUT_FILE)")" &
 else
-	@xdg-open $(BOOK_OUT) > /dev/null 2>&1 || open $(BOOK_OUT) &
+	@xdg-open $(BOOK_OUT_FILE) > /dev/null 2>&1 || open $(BOOK_OUT_FILE) &
 endif
 
 
+### Removes all built documentation
+clean_doc:
```

Comment:
I would change this to `clean-doc` for consistency.

Reply:
Agreed. I'll also change the other `run_pause` command to use a hyphen instead.